pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/kdefault/kustomization.go

```diff
@@ -61,7 +61,7 @@ namePrefix: {{ .ProjectName }}-
 #    someName: someValue
 
 resources:
-- ../crd
+#- ../crd
```

Comment:
It is not under the scope of this PR
The fix done in https://github.com/kubernetes-sigs/kubebuilder/pull/3626 is here otherwise that fails with the grafana sample